db/products.js

```
@@ -1,12 +1,12 @@
 const { client } = require('./index');
 
-async function createProduct({name, description, price, imageUrl, inStock, category}) {
+async function createProduct({name, description, price, imageURL='https://i.imgur.com/6CsuY8X.png', inStock, category}) {
     try {
         const { rows: [ product ] } = await client.query (`
-            INSERT INTO products(name, description, price, "imageUrl", "inStock", category)
+            INSERT INTO products(name, description, price, "imageURL", "inStock", category)
```

Have we tested what happens if we don't include some piece of data?  Maybe throw an error if so...?